### PR TITLE
BaSP-MR-311: Trainer and Member profiles

### PR DESCRIPTION
**Pull request by maurojjzz**
Styled Member and Trainer profile to show its own data. Create a form on each profile for the PUT petition.
Fix buttons position on Login component.
Hotfix on Subscription and Member's Form component. Now when you are an admin and want to edit a subscription or member the Date input shows an actual date.

### CODE REVIEW

**Comment by maurojjzz** [comment]
[Trainer profile and Member profile](https://trello.com/c/7DSPIr9S/311-trainer-profile-and-member-profile)

**Review by CristianLotorto — CHANGES_REQUESTED** [review]
when i try to add subscription the app crashes. I'm looking what can be :/. Take a look at that issue

**Review by Ginoboca1 — APPROVED** [review]

File: src/Components/Users/Member/profile/profile.module.css [hunk 1]
```diff
@@ -1,120 +1,77 @@
...
+  box-shadow: 0px 15px 25px -5px rgba(0, 0, 0, 0.75);
+}
+.editPhotoButton {
+  position: relative;
+  width: 30px;
+  height: 30px;
+  left: 52px;
+  bottom: 37px;
+  width: 30px;
+  height: 30px;
```
[the remark this answers is not in the document]
Comment (maurojjzz) [reply src/Components/Users/Member/profile/profile.module.css:38]: good catch

**Review by CristianLotorto — APPROVED** [review]
Nice work!

**Pull request merged**